spec.emu

```diff
@@ -46,20 +47,23 @@ contributors: Google, Ecma International
           <td>*"be"*</td>
           <td>*-&infin;*<sub>𝔽</sub></td>
           <td>*+&infin;*<sub>𝔽</sub></td>
+          <td>*-542*<sub>𝔽</sub></td>
```

Comment:
A couple of remarks about the epoch years.

I think they need to be mathematical values, not Number values, because they get subtracted from [[ISOYear]] which is a mathematical integer.

I also think there's an off-by-1 error in some of these, ISO year -542 seems to be Buddhist year 1, but the AO would return year 0. I think `gregory`, `japanese`, and all the other ones where the calendar year is the ISO year need to have 0 here. But `ethiopic` seems correct, ISO year 8 is the last of the previous era and ISO year 9 is calendar year 1. So please double-check them.

Reply:
This was a tricky one indeed. The ISO year 0 is the Gregorian year 1 and based on this I did some math which is where the off by one errors might be from. Will make the changes you mentioned.

Reply:
ISO year 0 is Gregorian year 1 BCE.